tests/backup/redeclaration-global2.gno

```diff
@@ -2,7 +2,6 @@ package main
 
 import (
 	"time"
-	"time"
```

Comment:
This file should be skipped.

Can we update the Makefile to ignore `tests` folder?

Reply:
Done